backend/src/test/java/de/neuefische/ffmjava221/teamprojekt/backend/weather/WeatherIntegrationTest.java

```diff
@@ -0,0 +1,42 @@
+package de.neuefische.ffmjava221.teamprojekt.backend.weather;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class WeatherIntegrationTest {
```

Comment:
We want to mock external apis in our test, like you did in your other test class. Either remove this one or add your mockwebserver